src/bun.js/VirtualMachine.zig

```diff
@@ -1953,7 +1957,9 @@ extern fn Bun__loadHTMLEntryPoint(global: *JSGlobalObject) *JSInternalPromise;
 pub fn reloadEntryPoint(this: *VirtualMachine, entry_path: []const u8) !*JSInternalPromise {
     this.has_loaded = false;
     this.main = entry_path;
+    this.main_resolved_path = .empty;
```

Comment:
I think this should deref first, because the previous string impl could only be used in js after this point
```suggestion
    this.main_resolved_path.deref();
    this.main_resolved_path = .empty;
```

Reply:
oh good point, fixed